### DISCUSSION TITLE
What are people doing about extension support? Sync support?

### DISCUSSION THREAD

**Discussion by fsociety3765** [VSCodium/vscodium#856] [Q&A]
Hi all,

I really want to make the switch to VSCodium full time but I am struggling when it comes to extension support. 

It seems like a lot of the extensions I use are not available on the marketplace for VSCodium. 

I know I can clone the repos, build from source and install manually but wondering if there is a better way to manage it? Doing this is OK for a couple of extensions but will start to become a pain quickly if I am having to do this for a lot.

Then there is the syncing aspect. I was using the Settings Sync (the one with like 160K downloads) extension until Microsoft implemented the sync feature directly. I've tried going back to the Settings Sync extension but it's a bit of a pain. If it can't find the extension it just sits there and does nothing. Can anyone recommend a better syncing extension for VSCodium?

Thanks,
D

**Comment by daiyam** [comment]
For syncing, you can try the extension [Sync Settings](https://open-vsx.org/extension/zokugun/sync-settings) (made by me, also available on VS Marketplace).

**Reply by beedell-roke** [reply]
@fsociety3765, [`discussions/793`](https://github.com/VSCodium/vscodium/discussions/793#discussioncomment-1140844) is a more focussed question on this topic.
